code/modules/mob/living/simple_animal/hostile/skeleton.dm

```diff
@@ -27,12 +27,18 @@
 	stat_attack = 1
 	gold_core_spawnable = 1
 	faction = list("skeleton")
-	see_invisible = SEE_INVISIBLE_MINIMUM
 	see_in_dark = 8
 	deathmessage = "collapses into a pile of bones!"
 	del_on_death = 1
 	loot = list(/obj/effect/decal/remains/human)
 
+/mob/living/simple_animal/hostile/skeleton/Login()
+	. = ..()
+	if (hud_used)
+		var/obj/screen/plane_master/lighting/L = hud_used.plane_masters["[LIGHTING_PLANE]"]
+		if (L)
+			L.alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+
```

Comment:
This boiler plate seems... complicated. Can't we have a var or a flag for it, that's checked in Life() or the super of Login()